### DISCUSSION TITLE
Save without comment

### DISCUSSION THREAD

**Discussion by unvalley** [bucketeer-io/bucketeer#848] [General]
Hi Buceteer Team,

Is there any plan to add a "Save without comment" feature (it should be "Save")? I don't really need to leave comments every time I save changes. We can check the event log and explain changes if necessary in other channels e.g. Slack. Not every save needs a comment, in my opinion.

**Comment by unvalley** [comment]
@cre8ivejp Hi, I think you are the current owner of bucketter. What do you think above post? :)

**Comment by cre8ivejp** [comment]
Thank you for your suggestion.

We have no plans to add that feature because we haven't yet received feedback on this topic before, but I understand your use case.
Because we don't support detailed audit logs yet,  people asked us to implement this comment feature so they can log more details about what exactly changed, etc.

We plan to improve the audit logs, including the change details in the new UI. When we do, we could leave the comment field as optional.

Either way, the changes must be checked before saving to avoid issues such as saving something by accident and affecting the end users.

I'll talk with the team to see what can be done before we release the new UI.

**Reply by unvalley** [reply]
Thanks for the reply.
When I was experimenting with various things in the development environment, I felt that comments were unnecessary. For example, if it is possible to define the environment of the project, I think it would be possible to make the comment mandatory only in the case of production.

I would be glad if you could consider this!

**Reply by cre8ivejp** [reply]
That makes totally sense to me.
I think we can work on that setting on the environment's page. 
So, for dev environments, you can just set it to off.
The only temporary issue is that only the Bucketeer team currently has access to the Projects and Environments Settings pages. So, you would need to ask you to enable or disable it for you. But, in the new UI, you will be able to manage it by yourself.

I will create an issue and see if we can work on this feature this month.
Thank you so much for your feedback!

Also, if you have any other feedback, please let me know so we can consider improving it in the new UI.
Even if it is a small thing, it would be very appreciated.

**Comment by cre8ivejp** [comment]
@unvalley, hi!

We decided to implement it as an option in the environment settings.
I will try to release this by the end of this month or early April.
I'll let you know when it is released.

**Reply by unvalley** [reply]
Cool, I can't wait the ship!

**Comment by cre8ivejp** [comment]
@unvalley, hi!
We have implemented the environment setting to make it optional.
Can you send me a private message telling me what environment you want me to make optional?

**Reply by unvalley** [reply]
Thank you! On a personal note, I just recently left the Mirainage team that uses Bucketeer.
I think the setting generally helps a lot.
